src/apps/chifra/pkg/cache/cache.go

```diff
@@ -28,17 +29,32 @@ func getCacheAndChainPath(chain string) string {
 func save(chain string, filePath string, content io.Reader) (err error) {
 	cacheDir := getCacheAndChainPath(chain)
 	fullPath := path.Join(cacheDir, filePath)
-	file, err := os.Create(fullPath)
-	if err != nil {
-		return
+
+	var file *os.File
+	if _, err = os.Stat(fullPath); os.IsNotExist(err) {
```

Comment:
There's a utility in our `file` package called `file.FileExists()` that returns a boolean.